### DISCUSSION TITLE
Add `custom_bmf2bf` option to `bmmodels`

### DISCUSSION THREAD

**Discussion by GidonFrischkorn** [venpopov/bmm#226] [Ideas]
When working with the `sdt` and `m3` models for `bmm` I thought about a generalizable way to specify when a custom `bmf2bf` method should be used. The issue is that parts of the `bmf2bf.bmmodel` method should be executed for all `bmmodels` but models such as the `sdt` and `m3` need to perform some additional specifications before other parts of the `bmmformula` should be converted into a `brmsformula`.

Maybe we can have a chat about this and brainstorm if there are other option. I just wanted to have a decision on this before sending you a pull request with changes that you would not agree with.

**Comment by venpopov** [comment]
I'm on vacation, will be back next week and we can discuss

**Reply by GidonFrischkorn** [reply]
I will be on vacation from next ween on and will be back in office the 5th of August. Maybe, we can meet the week I am back from vacation then.

**Comment by GidonFrischkorn** [comment]
Add `bmf2bf.default` to `bmmformula.R` to compile the first line of the `brmsformula` and have specific methods for `bmmodels` that require custom compilation of the first formula line.
